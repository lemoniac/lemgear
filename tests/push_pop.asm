ld SP, stack
ld BC, $aa44
push BC
pop DE
halt
nop
nop
nop
nop
nop
nop
stack:
