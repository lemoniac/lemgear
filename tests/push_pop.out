BC : AA44
DE : AA44

